sceptre/context.py

```diff
@@ -74,6 +77,7 @@ def __init__(self, project_path, command_path,
         self.output_format = output_format if output_format else ""
         self.no_colour = no_colour if no_colour is True else False
         self.ignore_dependencies = ignore_dependencies if ignore_dependencies is True else False
+        self.j2_extensions = j2_extensions if j2_extensions else ()
```

Comment:
I wonder if the default behaviour should be for Sceptre to look in a Jinja2 extensions directory? Just a thought.

Reply:
I would err on the side of caution and say that the default behaviour  should be as it is now - with no extensions loaded.

The problem with your suggestion of a `Jinja2 extensions directory` is that it's a lot more complex than this implementation. That would require dynamically loading modules from a directory (or directories) and that can get quite complex, it would also require defining a convention of how to name the exports from those files to then be passed to the jinja `Environment` constructor. 

The implementation I chose to go with separates sceptre from the above, the only prerequisite is that the module exists in `PYTHONPATH`